tests/test_rp_exec.py

```diff
@@ -387,7 +389,8 @@ def test_rp_raptor_staging(pilot_description, rp_venv):
         # (see https://docs.google.com/drawings/d/1q5ehxIVdln5tXEn34mJyWAmxBk_DqZ5wwkl3En-t5jo/)
 
         # Confirm that Master script is running (and ready to receive raptor tasks)
-        # WARNING: rp.Task.wait() *state* parameter does not handle tuples, but does not check type.
+        # WARNING: rp.Task.wait() *state* parameter does not handle tuples,
```

Comment:
@andre-merzky can you comment on whether this is resolved or has an ETA? Is there an issue I can watch?

Reply:
It is not resolved and has no ETA right now.